pkg/tsdb/elasticsearch/time_series_query_test.go

```diff
@@ -853,6 +853,80 @@ func TestExecuteTimeSeriesQuery(t *testing.T) {
 				"var1": "_count",
 			})
 		})
+
+		Convey("With aggregations that requires type casting", func() {
```

Comment:
There's a plan to move away from using `convey` and start using `testify`, asserting with `assert.` or `require.`.

Shouldn't be much trouble to do it - [check this out](https://github.com/grafana/grafana/blob/master/pkg/services/alerting/rule_test.go)